Marlin/src/lcd/extui_malyan_lcd.cpp

```diff
@@ -146,10 +146,10 @@ void process_lcd_eb_command(const char* command) {
 
       char message_buffer[MAX_CURLY_COMMAND];
       sprintf_P(message_buffer,
-        PSTR("{T0:%03.0f/%03i}{T1:000/000}{TP:%03.0f/%03i}{TQ:%03i}{TT:%s}"),
```

Comment:
This is the right thing to do - I'll test it but I expect it to just work, and it fixes the complaint when people don't select the right menu options when building.

Reply:
Sorry, I don't understand what you refer to by "This is the right thing to do."

You will note that this PR doesn't change any output or behavior. It just deals with the fact that `%03.0f` is always printed as "`   ?`" because Arduino doesn't implement `float` formatting in any of its `printf` functions.

Reply:
Arduino supports %f if you choose the correct option, right? This is how I've been helping people compile using the existing code - you choose a CRT lib with float support.  When I said "the right thing to do" I simply meant using an integer will 100% work, and since the malyan UI can't show decimal temperatures, there's no reason to not printf it as an int.
 
![arduinooptions](https://user-images.githubusercontent.com/32139633/62666214-4b596380-b937-11e9-8a6a-f623b84e68c4.png)

Reply:
Ah, ok. Your comment was on the old line, which is what confused me.

Reply:
Although some platforms may support `%f` we have to defer to the lowest common denominator. But, as it happens, `dtostrf` probably isn't any slower than `%f` would be.

Reply:
Hmm, well I guess we _could_ leave the Malyan code as it is, then. But as it is I don't think this change will make things worse.